tests/files/types_remote_test.php

```diff
@@ -112,22 +117,9 @@ public function test_get_max_file_size($max_file_size, $link, $expected = array(
 		$this->assertSame($expected, $file->error);
 	}
 
-	public function test_upload_timeout()
```

Comment:
Before we were able to specify a negative timeout and therefore hit the timeout limit before even uploading the file. With guzzle, the smallest timeout one can specify is 0.001s which is 1 ms. Anything below that or negative will be ignored, hence why this test is no longer possible. A 1 ms timeout still sometimes does not trigger the timeout during upload.